### PR TITLE
Fixing grammar issues in webpack example

### PR DESCRIPTION
**Pull request by stepharr**
**What kind of change does this PR introduce?**

Fix typo in Documentation 

**Did you add tests for your changes?**

Tests aren't needed

**If relevant, link to documentation update:**

N/A

**Summary**

Clarifying spelling in webpack example

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

File: examples/dll-app-and-vendor/0-vendor/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 This is the vendor build part.
 
-It's built separately from the app part. The vendors dll is only built when vendors has changed and not while the normal development cycle.
+It's built separately from the app part. The vendors dll is only built when the array of vendors has changed and not during the normal development cycle.
```
Comment (sokra) [inline examples/dll-app-and-vendor/0-vendor/README.md]: Update the `template.md` file too.

README.md is generated from this file.

Comment (stepharr) [reply examples/dll-app-and-vendor/0-vendor/README.md]: @sokra I don't see the text here to make that change: https://github.com/stepharr/webpack/edit/master/examples/dll-app-and-vendor/1-app/template.md.

Comment (montogeek) [reply examples/dll-app-and-vendor/0-vendor/README.md]: @stepharr He means this one https://github.com/stepharr/webpack/blob/master/examples/dll-app-and-vendor/0-vendor/template.md

**Comment by sokra** [comment]
Thanks

**Pull request merged**